receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -82,14 +82,14 @@ func (s *splunkScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 
 // Each metric has its own scrape function associated with it
 func (s *splunkScraper) scrapeLicenseUsageByIndex(ctx context.Context, now pcommon.Timestamp, errs *scrapererror.ScrapeErrors) {
-	var sr searchResponse
-	// Because we have to utilize network resources for each KPI we should check that each metrics
-	// is enabled before proceeding
-	if !s.conf.MetricsBuilderConfig.Metrics.SplunkLicenseIndexUsage.Enabled {
+	if !s.conf.MetricsBuilderConfig.Metrics.SplunkLicenseIndexUsage.Enabled || !s.splunkClient.isConfigured(typeCm) {
 		return
 	}
+	ctx = context.WithValue(ctx, endpointType("type"), typeCm)
+	// Because we have to utilize network resources for each KPI we should check that each metrics
+	// is enabled before proceeding
```

Comment:
these changes will need to be made on the new indexer search scrapes also I assume? If so mind adding those to the newly merged searches from last PR?